doc/DOCUMENTATION.org

```diff
@@ -3251,39 +3255,43 @@ To search in a project see [[#searching-in-a-project][project searching]].
+| ~SPC p d~   | find directory                                                                         |
+| ~SPC p D~   | open project root in =dired=                                                           |
+| ~SPC p e~   | edit dir-locals.el                                                                     |
+| ~SPC p f~   | find file                                                                              |
+| ~SPC p F~   | find file based on path around point                                                   |
+| ~SPC p g~   | find tags                                                                              |
+| ~SPC p G~   | regenerate the project's =etags= / =gtags=                                             |
+| ~SPC p h~   | find file                                                                              |
+| ~SPC p I~   | invalidate the projectile cache                                                        |
+| ~SPC p k~   | kill all project buffers                                                               |
+| ~SPC p o~   | run =multi-occur=                                                                      |
+| ~SPC p p~   | switch project                                                                         |
+| ~SPC p r~   | open a recent file                                                                     |
+| ~SPC p R~   | replace a string                                                                       |
+| ~SPC p t~   | open =NeoTree= in =projectile= root                                                    |
+| ~SPC p T~   | test project                                                                           |
+| ~SPC p v~   | open project root in =vc-dir= or =magit=                                               |
+| ~SPC /~     | search in project with the best search tool available                                  |
+| ~SPC s p~   | see [[#searching-in-a-project][searching in a project]]                                                             |
+| ~SPC s a p~ | run =ag=                                                                               |
+| ~SPC s g p~ | run =grep=                                                                             |
+| ~SPC s k p~ | run =ack=                                                                              |
+| ~SPC s t p~ | run =pt=                                                                               |
+| ~SPC s r p~ | run =rg=                                                                               |
+| ~SPC f y C~ | show and copy current file relative path with line and column number in the minibuffer |
+| ~SPC f y D~ | show and copy current directory relative path in the minibuffer                        |
+| ~SPC f y L~ | show and copy current file relative path with line number in the minibuffer            |
+| ~SPC f y Y~ | show and copy current file relative path in the minibuffer                             |
```

Comment:
Initially, I thought these bindings should be documented with the absolute-path counterparts.  Maybe clarify "relative to the project root"?